udata/tests/frontend/test_csv.py

```diff
@@ -59,7 +59,7 @@ class Meta:
 
     title = factory.LazyAttribute(lambda o: faker.sentence())
     description = factory.LazyAttribute(lambda o: faker.paragraph())
-    tags = factory.LazyAttribute(lambda o: [faker.word() for _ in range(1, randint(1, 4))])
+    tags = factory.LazyAttribute(lambda o: [faker.tag() for _ in range(1, randint(1, 4))])
```

Comment:
Why not use `faker.tags()` here?

Reply:
I didn't want to change the code, but I sure could, let me do that quickly ;)